sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceVersion.java

```diff
@@ -8,15 +8,16 @@
 
 /** Service version of WebPubSubService. */
 public enum WebPubSubServiceVersion implements ServiceVersion {
-    /** Enum value 2021-10-01. */
-    V2021_10_01("2021-10-01");
+    /** Enum value 2022-11-01. */
+    V2022_11_01("2022-11-01");
```

Comment:
We should never remove a service version if the library GA'd with that service version. We should keep V2021_10_01 and make V2022_11_01 the latest.